src/rewriter/rewrite_db_proof_cons.h

```diff
@@ -0,0 +1,66 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
```

Comment:
```suggestion
 * Copyright (c) 2009-2023 by the authors listed in the file AUTHORS
```
This file has been waiting for a while. :)

Reply:
Yes it has, :-)